<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorWidgetForm</class>
 <widget class="QWidget" name="MonitorWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="monitorStopAllScanButt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop All Scan</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monitorStopCurrentScanButt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Current Scan</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monitorDetailsScanButt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Scan Details</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/traceroute.png</normaloff>:/images/images/traceroute.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="scanProgressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="scanMonitor">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Host Thread</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Scan Options</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>State</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
